azkaban-common/src/main/java/azkaban/metrics/DummyContainerMetricsImpl.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2021 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package azkaban.metrics;
+
+import azkaban.utils.Props;
+import org.apache.log4j.Logger;
+
+/**
+ * No-op implementation of {@link ContainerMetrics} used when dispatch method is not
```

Comment:
Not sure if this class is really necessary. Please refer to the comment in the AzkabanWebServerModule.